temba/channels/migrations/0050_add_junebug_channel_type.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Just merged from us because the migration squashing wasn't on /rapidpro yet and redoing that wasn't in the cards. Shouldn't affect this PR except that this migration should be 0058 and depend on 0057